modules/storage/src/main/kotlin/edu/stanford/spezi/modules/storage/key/PreferenceKey.kt

```diff
@@ -0,0 +1,20 @@
+package edu.stanford.spezi.modules.storage.key
+
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.booleanPreferencesKey
+import androidx.datastore.preferences.core.byteArrayPreferencesKey
+import androidx.datastore.preferences.core.doublePreferencesKey
+import androidx.datastore.preferences.core.floatPreferencesKey
+import androidx.datastore.preferences.core.intPreferencesKey
+import androidx.datastore.preferences.core.longPreferencesKey
+import androidx.datastore.preferences.core.stringPreferencesKey
+
+sealed class PreferenceKey<T>(val key: Preferences.Key<T>) {
```

Comment:
Let's mark key as `internal val key`, otherwise consumers of this module would need to add datastore dependency